### PR TITLE
Unify WindowDesc and PendingWindow

### PR DESCRIPTION
**Pull request by pyroxymat**
Solves issue #536

### CODE REVIEW

File: druid/src/win_handler.rs [hunk 1]
```diff
@@ -85,21 +85,22 @@ struct Inner<T> {
...
+    pending: HashMap<WindowId, WindowDesc<T>>,
     windows: HashMap<WindowId, Window<T>>,
 }
 
 impl<T> Windows<T> {
     fn connect(&mut self, id: WindowId, handle: WindowHandle) {
         if let Some(pending) = self.pending.remove(&id) {
-            let win = pending.into_window(id, handle);
+            let win = Window::new(id, handle, pending);
+            // let win = pending.into_window(id, handle);
```
Comment (cmyr) [inline druid/src/win_handler.rs]: :grin:

File: druid/src/window.rs [hunk 2]
```diff
@@ -53,27 +46,14 @@ pub struct Window<T> {
...
+impl<T> Window<T> {
+    pub(crate) fn new(id: WindowId, handle: WindowHandle, desc: WindowDesc<T>) -> Window<T> {
         Window {
             id,
-            root,
-            size: Size::ZERO,
-            title,
-            menu,
+            root: WidgetPod::new(desc.root),
+            size: desc.size.unwrap_or(Size::ZERO),
```
Comment (cmyr) [inline druid/src/window.rs]: I think we want to keep this as `Size::ZERO`; the size in the window description is passed to the window builder, but is not the actual size of the window itself; we will get that back from the platform once the window connects.

Comment (pyroxymat) [reply druid/src/window.rs]: I changed it back to Size::ZERO

**Comment by cmyr** [comment]
oh one other thing: does the multiwin example still work after this change? I'll check in a few minutes but feel free to check first if you're around.

**Comment by cmyr** [comment]
thanks!

**Pull request merged**